lib/ssh/src/ssh.erl

```diff
@@ -225,7 +229,7 @@ Opaque data type representing a connection between a client and a server
 Returned by the functions [`connect/2,3,4`](`connect/3`) and
 [`ssh_sftp:start_channel/2,3`](`ssh_sftp:start_channel/2`).
 """.
--doc(#{title => <<"Other data types">>}).
+-doc(#{title => <<"Other Data Types">>}).
```

Comment:
Could we perhaps find better categories than "Other" ?
Maybe not all now other belong in the same category!?

Reply:
I don't have an idea now. But I removed the title so we just have a generic `Types` category with them.